man/systemd-tty-ask-password-agent.xml

```diff
@@ -92,11 +92,11 @@
       </varlistentry>
 
       <varlistentry>
-        <term><option>--console</option></term>
+        <term><option>--console</option><optional>=<replaceable>DEVICE</replaceable></optional></term>
 
-        <listitem><para>Ask question on
-        <filename>/dev/console</filename> instead of querying the user
-        on the calling TTY. </para></listitem>
+        <listitem><para>Ask question on <replaceable>DEVICE</replaceable> instead of querying the user on the
```

Comment:
maybe say "Ask question on TTY <replaceable>…", i.e. clarify what kind of device this is, i.e. a tty. (not that people might think it might be an input device, or DRM or fb or whatever)